tests/test_federation.py

```diff
@@ -27,13 +27,13 @@ def setUp(self):
         user_id = UserID("us", "test")
         our_user = Requester(user_id, None, False, None, None)
         room_creator = self.homeserver.get_room_creation_handler()
-        room = ensureDeferred(
+        room_d = ensureDeferred(
```

Comment:
`room_d` to me implies a `Dict`, but this is a `Tuple[Dict, int]`.

Reply:
This is actually a deferred

Reply:
Oh, I see now.